lldb/source/API/CMakeLists.txt

```diff
@@ -323,12 +332,24 @@ foreach(header
     COMMENT "LLDB headers: stage LLDB headers in include directory")
 
   list(APPEND lldb_staged_headers ${staged_header})
+
+  if (LLDB_BUILD_FRAMEWORK)
+    set(input_header ${staged_header})
```

Comment:
Seems like you could just use `${staged_header}` directly? I don't think this adds much.